src/pool-bin/libraries/BinPool.sol

```diff
@@ -396,10 +398,21 @@ library BinPool {
         amountsLeft = params.amountIn;
 
         for (uint256 i; i < params.liquidityConfigs.length;) {
```

Comment:
![Screenshot 2024-05-21 at 10 27 37](https://github.com/pancakeswap/pancake-v4-core/assets/133624774/75b4c5e1-7e2c-486f-8a97-50206abd8983)

did a check, seems like moving declaration outside of `for loop` could save gas, could you give it a try? thanks

Reply:
checked and it did help to save gas.. thx for reminding